Designing a light microscopy experiment episode (#41)
HealthBioscienceIDEAS/microscopy-novice: episodes/designing-a-light-microscopy-experiment.md

---
title: 'Designing a light microscopy experiment'
teaching: 30
exercises: 10
---

:::::::::::::::::::::::::::::::::::::: questions 

- What are the key factors to consider when designing a light microscopy 
experiment?

::::::::::::::::::::::::::::::::::::::::::::::::

::::::::::::::::::::::::::::::::::::: objectives

- Explain the main steps of designing a light microscopy experiment

- Describe a few examples of common microscopy methods e.g. widefield, confocal

- Explain factors to consider when choosing a microscopy technique

::::::::::::::::::::::::::::::::::::::::::::::::

In previous episodes we focused on the fundamentals of how images are stored and 
displayed, with Napari as our main example. In the next few episodes, we will 
instead focus on how to design an experiment using light microscopy and how to 
analyse the resulting image data.

## Steps to designing a light microscopy experiment

> As we've said throughout the course, there is no one 'right' way of handling 

Author: Good point - I'll change it to analysing

image data. It depends on the research question and project in hand, with 
different solutions being best in different situations. The same holds true for 
designing a light microscopy experiment - here we'll discuss some general 
guidelines, but the specifics will depend on your particular research project.

For the sake of this episode, let's imagine a scenario where we are 
investigating the effects of a specific chemical on cells grown in culture.
What steps would we take to design a light microscopy experiment for this?

A general workflow could be:

1. Define your research question

2. Define what you need to observe to answer that question

3. Define what you need to measure to answer that question

4. Choose a light microscopy method that fits your data needs

5. Choose acquisition settings that fit your data needs

## Define your research question

First, it's essential to clearly define your research question. What do you want 
to find out from your microscopy experiment?

For example, for our chemical example, the question could be: Does the chemical 

> affect the number, size or shape of cell nuclei over time?

Reviewer: This is a very general research question, it's not really a hypothesis that can be tested with a microscopy experiment, I think maybe it's a research theme? For the purposes of this lesson that is probably a good thing as it allows you to be quite general in your discussion and enables the students to think of something more specific. However maybe it's worth building on it here. For me a research question would be more specific "Adding chemical X results in an increase in the number of cell nuclei over N minutes."

Author: I'll expand this section to give some more specific (and realistic!) examples of research questions

segment the nuclei and measure their volume and sphericity. We'll look at some 
techniques for this kind of analysis in the [manual segmentation](FIXME.md), 
[thresholding](FIXME.md) and [instance segmentation](FIXME.md) episodes.

:::::::::::::::::::::::::::::::::

::::::::::::::::::::::::::::::::::::::::::::::::

## Choose a light microscopy method

For the final part of this episode, let's look at how we can choose a light 
microscopy method for our experiment. We'll look at the last step (choosing 
acquisition settings) in the next episode.

Often the first step in choosing a light microscopy technique is discussing with 
other members of your research group or department. What are most of them using, 
for what kind of experiments? If you are working with a microscopy core facility, 
then the people who work there are also a great source of advice and 
recommendations. Finally, you can also explore the literature for research 
questions and experiments similar to yours to see which methods are used most 
often.

There's a very wide variety of light microscopes available (from many different 
manufacturers) that are specialised for different kinds of samples. We'll take a 

> brief look at some popular options below, but bear in mind that there are many 

Reviewer 2: that there are far too many to discuss them all in this lesson.

more!

### Widefield

In a widefield microscope, the entire sample is illuminated by the light source 
at once. The light source can illuminate the sample from below (in an upright 
microscope) or from above (in an inverted microscope).

In it's simplest form, a widefield microscope can be used for 'brightfield 
microscopy'. This is simply where the sample is illuminated by a bright light 
from one side, and then imaged from the other. The issue with this method is 

> that it often produces low-contrast images, where biological structures are 

Reviewer 2: where it is difficult to see (define?) biological structures.

hard to see. This is because biological structures are often quite transparent - 

> they don't absorb much light or differ much in their density. For this reason, 

Reviewer 2: OK, this makes it clearer to me why fluorescent is needed. I wonder if some of this stuff might be better earlier? At least a signpost to this discussion?

contrast agents/staining are often used to increase contrast (i.e. the addition 
of dyes/chemicals that bind to specific structures).

![The image above is Napari's Skin (RGB) sample image - it is a brightfield 
image of a hematoxylin and eosin stained slide of dermis and epidermis.
](fig/skin.png){alt="Screenshot of Napari's Skin sample image" width='70%'}

To increase contrast (especially for unstained samples), widefield microscopes 
often support ['phase-contrast'
](https://www.microscopyu.com/techniques/phase-contrast/introduction-to-phase-contrast-microscopy) 
or ['DIC - Differential Interference Contrast'
](https://www.leica-microsystems.com/science-lab/microscopy-basics/differential-interference-contrast-dic/).

> 

Reviewer 2: Is it worth one sentence explaining what these techniques do to improve contrast?

![The image above is a phase gradient contrast image of some SH-SY5Y cells 
(ZEISS Microscopy, [CC BY 2.0](https://creativecommons.org/licenses/by/2.0), 
via [Wikimedia Commons
](https://commons.wikimedia.org/wiki/File:SH-SY5Y_cells,_transmitted_light_phase_gradient_contrast_microscopy_with_ZEISS_Celldiscoverer_7_(30614936722).jpg)
)](fig/phase-contrast.jpg){alt="Phase gradient contrast image of SH-SY5Y 
cells" width='70%'}

![The image above is a DIC image of some yeast cells (_Saccharomyces cerevisiae_) 
from [Wikimedia Commons
](https://commons.wikimedia.org/wiki/File:S_cerevisiae_under_DIC_microscopy.jpg)
](fig/dic-example.jpg){alt="DIC image of some yeast cells - Saccharomyces 
cerevisiae" width='50%'}

Widefield microscopes can also be used for fluorescence microscopy. In 
fluorescence microscopy, fluorescent labels are used that target specific 
features (like the nucleus or cell membrane). These labels are excited by a 
specific wavelength of light (the excitation wavelength), and then emit light of 
a longer wavelength (the emission wavelength). By illuminating the sample with 
light of the excitation wavelength, then detecting light of the emission 

> wavelength, a fluorescence microscope can image the biological structures that 

Reviewer 2: Are they any drawbacks to fluorescent? Such as, wouldn't you need to have the fluorophore of interest (and how costly is it to purchase? Do microscopes need to have the right excitation wavelength (or do they all have the required lasers to do most fluorophores)?

Author: I've added a few more details now - there are definitely drawbacks to requiring fluorescence! Acquiring good fluorescent labels can be challenging + they have to match the capabilities of your microscope

the label is bound to.

![The image above is a fluorescence microscopy image of some LLC-PK1 cells 
(ZEISS Microscopy, [CC BY 2.0](https://creativecommons.org/licenses/by/2.0), 
via [Wikimedia Commons
](https://commons.wikimedia.org/wiki/File:Mitotic_LLC-PK1_cells,_fluorescence_microscopy_(23700644352).jpg)
)](fig/fluorescence-example.jpg){alt="Fluorescence microscopy image of some 
LLC-PK1 cells" width='70%'}

> The main issue with widefield microscopes is that, due to the whole sample being 

Reviewer 2: So this is essentially compressing the third dimension down to two?

Author: I think so - you're detecting light from the focal plane + areas that are illuminated above and below it. So you're definitely incorporating some 3D information into your 2D image.

illuminated, it can produce rather blurry images (especially for thicker 
samples). For example, in widefield fluorescence microscopy, fluorophores 
throughout the entire sample will be emitting light. This means that the 
resulting image will represent light from the focal plane, and also from 
out-of-focus planes above and below it. This can result in details being 
obscured by out-of-focus light.

### Confocal

Confocal microscopes are capable of 'optical sectioning' meaning they can detect 
light only from the plane of focus and exclude light from out-of-focus planes. 
This is extremely useful, especially for thick samples, to achieve clear images 
where details aren't obscured by out-of-focus light.

There are many different types of confocal microscopes, but one of the most 

> common is: laser scanning confocal microscopes. These are also known as point 

Reviewer 2: ```suggestion
common is laser scanning confocal microscopes. These are also known as point 
```

scanning confocal microscopes. In this microscope, a laser is focused onto a 
single point of the specimen at a time (rather than illuminating the entire 
sample). The point of illumination is then moved across the sample in a raster 
pattern (i.e. line by line), with light detected point by point. This gradually 
builds up the final image.

Confocal microscopes are typically used for fluorescence microscopy, especially 

> when it is necessary to image the shape of structures in full 3D. Bear in mind 
> that their advantages don't come without other disadvantages! For example, 

Reviewer 2: ```suggestion
when it is necessary to image the shape of structures in full 3D. However, this for of imaging also comes with some disadvantages. For example, 
```

confocal microscopes will generally be more complex to use and slower to acquire 
images than a widefield microscope. Although there are other types of confocal 

> that are faster than laser scanning systems, such as 'spinning disc confocal 

Reviewer: Would a link to more information about spinning disc confocal things be useful?
Pros: I'm curious, 
Cons: If there was a hyperlink I'd follow it then get distracted from the lesson aims.

Author: I'll add a link to a review paper on confocal systems (laser scanning + spinning disc) - it adds a lot of interesting extra info on these microscopes!

> microscopes'.

Reviewer 2: Are they faster because they acquire the data quicker than a raster procedure does?

Author: I've added some extra information to the text now. Spinning disc microscopes are faster as they split the laser into many hundreds of focused beams that then move across the sample together. So it can image many locations at once vs a standard laser scanning confocal that only has one beam.

![The image above is Napari's Kidney (3D + 3Ch) sample image. This was acquired 
with confocal fluorescence microscopy.](fig/confocal-example.png){alt="Screenshot 
of Napari's Kidney (3D + 3Ch) sample image" width='60%'}

### Super-resolution

> As there are so many [different 'super-resolution' methods

Reviewer 2: I would move this first sentence to the end of the paragraph. And maybe put some examples of scale as to what super-resolution can do and what is the theoretical limit that it is breaking in terms of distance.

Author: I've added some values for the diffraction limit + maximum resolutions of standard vs super-resolution microscopes

usually comes at the cost of increased complexity in experiment/microscope setup.

### Choosing a method

Choosing a light microscopy method is about finding the simplest and fastest 
approach that can address your research question. For example, does it provide 
the resolution you need? Can it provide good images of thick vs thin samples? 
Does it image a large enough area? Does it image fast enough to provide the 
temporal resolution you need?... The online [bioimaging guide
](https://www.bioimagingguide.org/welcome.html) provides a [useful flowchart
](https://www.bioimagingguide.org/02_Sample_acquisition/Picking.html) to give 
you some ideas - as well as links to many other great microscopy resources.

If your research question can be solved by a simpler and less costly method, 
then usually this is the way to go! It's all about choosing the right tool for 
the job - different approaches will be best for different research questions. 
For example, you could count cell number with a high-end super-resolution 
microscope, but this is also possible with a standard widefield which will be 
simpler and faster. In general, only move to more complex techniques when your 

> research question really requires it.

Reviewer 2: Sounds like good advice for statistical models and image processing algorithms as well.

::::::::::::::::::::::::::::::::::::: challenge 

## Which microscope?

Which light microscopes could be used to answer our research question - 'Does the 
chemical affect the number, size or shape of cell nuclei over time?'

Think about the points above and make a list of potential answers. It may be 
best to discuss in a group, so you can share different ideas.

:::::::::::::::::::::::: solution 

Again, there's no one solution here - it depends on which elements of nucleus 
shape and size we're focusing on. If we're happy with 2D measures, then 
widefield fluorescence microscopy would be a great option to allow large numbers 
of nuclei to be imaged over time.

If we're more interested in the precise 3D volume and shape, then confocal 
fluorescence microscopy would be a better fit. We'd have to be careful with how 

> often we wanted to image the cells vs the speed of acquisition though.

Reviewer 2: As we said earlier we wanted to do some live tracking of the cells (or did I imagine that). If we did, it's worth mentioning this again here I think.

> Super-resolution options wouldn't be required here. Cell nuclei can be easily 

Reviewer 2: Again, may be worth highlighting what sort of resolution we are talking about in terms of the technique and the size of the nuclei

visualised with standard widefield and confocal approaches. There's no need to 
introduce any extra complexity, as our research question doesn't require that 
level of resolution.

:::::::::::::::::::::::::::::::::

::::::::::::::::::::::::::::::::::::::::::::::::

::::::::::::::::::::::::::::::::::::: keypoints 

> - A general workflow for designing a light microscopy experiment could include:

Reviewer 2: I think these are so essential that it should be "should include" rather than "could include". Presumably more steps might be present in a more advanced workflow?

  - Define your research question
  - Define what you need to observe to answer that question
  - Define what you need to measure to answer that question
  - Choose a light microscopy method that fits your data needs
  - Choose acquisition settings that fit your data needs
  
- There are many different types of light microscope - including widefield and 
confocal

> - You should choose the simplest method that allows you to address your 

Reviewer 2: ```suggestion
- You should choose the simplest methods (in acquisition, processing, and statistical analysis) that allows you to address your a
```

analyse the resulting image data.

## Steps to designing a light microscopy experiment

As we've said throughout the course, there is no one 'right' way of analysing 
image data. It depends on the specific images and research question, with 
different solutions being best for different situations. The same holds true for 
designing a light microscopy experiment - here we'll discuss some general 
guidelines, but the specifics will depend on your particular research project.

For the sake of this episode, let's imagine a scenario where we are 
investigating the effects of a specific chemical on cells grown in culture.
What steps would we take to design a light microscopy experiment for this?

A general workflow could be:

1. Define your research question

2. Define what you need to observe to answer that question

3. Define what you need to measure to answer that question

4. Choose a light microscopy method that fits your data needs

5. Choose acquisition settings that fit your data needs

## Define your research question

Clearly defining your research question is the essential starting point. What do 
you want to find out from your microscopy experiment?

> For example, for our chemical example, the main aim could be: Does the chemical 

Reviewer 2: ```suggestion
For our chemical example, the main aim could be: Does the chemical 
```

affect the number, size or shape of cell nuclei over time? 

Then we could form more specific research questions e.g.

> - Does the addition of the chemical, result in an increase in the number of cell 
> nuclei over ten hours?
> 
> - Does the addition of the chemical, result in an increase in the average 
> nucleus diameter over ten hours?
> 
> - Does the addition of the chemical, result in an increase in the roundness of 
> cell nuclei over ten hours?

Reviewer 2: ```suggestion
- Does the addition of the chemical result in an increase in the number of cell 
nuclei over ten hours?

- Does the addition of the chemical result in an increase in the average 
nucleus diameter over ten hours?

- Does the addition of the chemical result in an increase in the roundness of 
cell nuclei over ten hours?
```

:::::::::::::::::::::::::::::::::

::::::::::::::::::::::::::::::::::::::::::::::::

## Define what you need to measure

Once we have a good idea of what we need to observe, we need to clearly define 
what measurements are required. 

- What do you need to quantify from your images to answer your question? E.g. 
cell length, nucleus volume, the number of cells with a specific fluorescent 
label...

- How will you quantify them? E.g. which image processing software? What kind of 
analysis methods?

- How many objects/events do you need to quantify?

As the second point indicates, it's always worthwhile to consider how you will 
quantify your images before you collect them. This will make the image analysis 
steps much faster and ensure that you are collecting images optimised for the 
given method. We'll look at image processing methods more in later episodes.

For the last point, it's important to consider any statistical tests you will 
use to answer your research question, and the sample size required to give them 
sufficient statistical 'power'. Statistical power is the likelihood of a 
statistical test detecting an effect when one is actually present. For example, 

> consider testing if the number of cells is significantly different when grown 
> with and without our chemical of interest. If there really is a difference, a 

Reviewer 2: ```suggestion
consider testing if there is a difference of X% in the number of cells  when grown 
with and without our chemical of interest. If there really is a difference, a 
```
Subtle change - any level of difference can be significant with the right sample size. What we want is a relevant effect size or difference in the number of cells and make sure that we are powered to detect that level of effect size.
